collect/collect.go

```diff
@@ -35,6 +35,45 @@ type InventoryCollection struct {
 	TLSKey             string
 	AuthEnabled        bool
 	Signer             jose.Signer
+	MetaData           *metaData
```

Comment:
Any reason this is an exported field? I general I would always try to keep the scope as narrow as possible

Reply:
Fixed throughout the whole struct